src/client/components/board/board.test.tsx

```diff
@@ -0,0 +1,110 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Board from './board';
+import { DEFAULT_TURN_DURATION } from '../../../utils/constants';
+import { DEFAULT_GAME_MODE } from '../../../utils/GameMode';
+
+jest.mock('../model/model.jsx');
+
+const G = {
+  dealt: [],
+  players: {
+    0: ['T3', 'T4', 'T5'],
+  },
+  order: [0, 1, 2],
+  scores: [0, 0, 0],
+  identifiedThreats: {},
+  selectedDiagram: 0,
+  selectedComponent: '',
+  threat: {
+    modal: false,
+  },
+  passed: [],
+  suit: 'T',
+  round: 1,
+  startingCard: 'T3',
+  gameMode: DEFAULT_GAME_MODE,
+  turnDuration: DEFAULT_TURN_DURATION,
+  turnFinishTargetTime: Date.now() + DEFAULT_TURN_DURATION * 1000,
+};
+const ctx = {
+  actionPlayers: [0, 1, 2],
+};
+Board.prototype.componentDidMount = jest.fn();
```

Comment:
Why is this being mocked?

Reply:
Was mocked in the old version of the test and I did not put much thought into it. Removed it.